client/src/definitions/datasetFilters.ts

```diff
@@ -1,7 +1,9 @@
 import type { SelectOption } from "./form";
+import type { Organization } from "./organizations";
 import type { Tag } from "./tag";
 
 export type DatasetFiltersInfo = {
+  organizationSiret: Organization[];
```

Comment:
```suggestion
  organizations: Organization[];
```
A en voir le typage ce tableau ne contient pas que les siret si ?

Reply:
En effet, il contient bien un array `Organizations[]`. Mais c'est la même idée que le filtre `tagId` qui contient un array `Tag[]`. Le nom du filtre correspond au champ = la valeur qui va être sélectionnée. Mais le contenu peut être différent, en l'occurrence parce qu'on a besoin d'un ID (la valeur envoyée au backend) et d'un name (le label à afficher).